frontend/rolecall/src/app/homepage/dashboard.component.scss

```diff
@@ -1,5 +1,5 @@
-@import "../../styles/colors";
-@import "../../styles/constants";
+@import '../../styles/colors';
```

Comment:
Looks like we have conflicting style rules on double-quotes in CSS files, can we change this back to match Google3 style rules?